tests/testthat/test-lsbinaryclassification.R

```diff
@@ -167,11 +167,16 @@ test_that("Probability positive plot matches", {
   jaspTools::expect_equal_plots(testPlot, "probability-positive-data", dir="LSbinaryclassification")
 })
 
-test_that("Receiving Operating Characteristic Curve plot matches", {
-  plotName <- results[["results"]][["plots"]][["collection"]][["plots_plotROC"]][["data"]]
-  testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
-  jaspTools::expect_equal_plots(testPlot, "receiving-operating-characteristic-curve-data", dir="LSbinaryclassification")
-})
+{
+  #TODO: Use Don's vdiffr version
+  skip_on_os(c("mac", "linux"))
+  test_that("Receiving Operating Characteristic Curve plot matches", {
```

Comment:
```suggestion
  test_that("Receiving Operating Characteristic Curve plot matches", {
    skip_on_os(c("mac", "linux"))
```
otherwise you are skipping every test after this one

Reply:
oh shoot, thanks for noticing. I thought that using ``{...}`` would suffice